nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -480,6 +519,11 @@ in {
               'trusted_proxies' => ${writePhpArrary (c.trustedProxies)},
               ${optionalString (c.defaultPhoneRegion != null) "'default_phone_region' => '${c.defaultPhoneRegion}',"}
             ];
+
+            $EXTRACONFIG = json_decode('${builtins.toJSON cfg.extraOptions}', true);
+
+            array_push($CONFIG, $EXTRACONFIG);
```

Comment:
Is `array_push` the thing you actually need here?

```
>>> $foo=[1,2,3,4];
=> [
     1,
     2,
     3,
     4,
   ]
>>> $b=[4,5,6,7];
=> [
     4,
     5,
     6,
     7,
   ]
>>> array_push($foo,$b);
=> 5
>>> $foo
=> [
     1,
     2,
     3,
     4,
     [
       4,
       5,
       6,
       7,
     ],
   ]

```

Reply:
resolved in a different way (see your other thread further down)